dx-1126 note on deprecated href operations (#599)
SwedbankPay/developer.swedbankpay.com: _includes/payment-order-other-features.md

determine the available operations before the initial purchase.
A list of possible operations and their explanation is given below.

> {% include alert.html type="informative" icon="info" header="Note on deprecated
> operations." body="Several operations have now the status deprecated.
> These operations are:
> `create-capture`
> `create-cancel`
> `create-reversal`
> Because they are still in use by the existing users, they will not be removed from
> the documentation yet. New users should not use these operations and these
> operations will be marked with `DEPRECATED DO NOT USE!`." %}

Reviewer: ```suggestion
{% include alert.html type="informative" icon="info" header="Deprecated
Operations." body="Payment instrument-specific operations are passed
through Payment Order. These can be recognized by not having
`paymentorder` in the `rel` value. They will be described and marked as
deprecated in the operation list below." %}
```

            "rel": "view-paymentorder",
            "contentType": "application/javascript"
        },

>         {   `DEPRECATED DO NOT USE!`

Reviewer: `create-paymentorder-capture` is not deprecated. It is `create-capture` (which is missing from the operations list) that is deprecated.

```suggestion
        {
```

            "method": "POST",
            "href": "{{ page.api_url }}/psp/paymentorders/{{ page.payment_order_id }}/captures",
            "rel": "create-paymentorder-capture",
            "contentType": "application/json"
        },

>         {   `DEPRECATED DO NOT USE!`

Reviewer: `create-paymentorder-cancel` is not deprecated. It is `create-cancel` (which is missing from the operations list) that is deprecated.

```suggestion
        {
```

            "method": "POST",
            "href": "{{ page.api_url }}/psp/paymentorders/{{ page.payment_order_id }}/cancellations",
            "rel": "create-paymentorder-cancel",
            "contentType": "application/json"
        },

>         {   `DEPRECATED DO NOT USE!`

Reviewer: `create-paymentorder-reversal` is not deprecated. It is `create-reversal` (which is missing from the operations list) that is deprecated.

```suggestion
        {
```

            "rel": "failed-paymentorder",
            "contentType": "application/problem+json"
        }

>         {`DEPRECATED DO NOT USE!`

Reviewer: ```suggestion
        {
            // Deprecated operation. Do not use!
```

            "method": "POST",

>             "href": "{{ page.api_url }}/psp/paymentorders/{{ page.payment_order_id }}/captures",

Reviewer: I think the `href` points directly to the payment instrument. Please investigate.

```suggestion
            "href": "{{ page.api_url }}/psp/creditcard/{{ page.payment_id }}/captures",
```

            "rel": "create-capture",
            "contentType": "application/json"
        },

>         {`DEPRECATED DO NOT USE!`

Reviewer: ```suggestion
        {
            // Deprecated operation. Do not use!
```

            "method": "POST",

>             "href": "{{ page.api_url }}/psp/paymentorders/{{ page.payment_order_id }}/cancellations",

Reviewer: I think the `href` points directly to the payment instrument. Please investigate.

```suggestion
            "href": "{{ page.api_url }}/psp/creditcard/{{ page.payment_id }}/cancellations",
```

            "rel": "create-cancel",
            "contentType": "application/json"
        },

>         {`DEPRECATED DO NOT USE!`

Reviewer: ```suggestion
        {
            // Deprecated operation. Do not use!
```

            "method": "POST",

>             "href": "{{ page.api_url }}/psp/paymentorders/{{ page.payment_order_id }}/reversals",

Reviewer: I think the `href` points directly to the payment instrument. Please investigate.

```suggestion
            "href": "{{ page.api_url }}/psp/creditcard/{{ page.payment_id }}/reversals",
```

| `update-paymentorder-updateorder` | Updates the order with a change in the `amount` and/or `vatAmount`.                                                                                                                                                                                                            |
| `redirect-paymentorder`           | Contains the URI that is used to redirect the consumer to the Swedbank Pay Payments containing the Payment Menu.                                                                                                                                                               |
| `view-paymentorder`               | Contains the JavaScript `href` that is used to embed the Payment Menu UI directly on the webshop/merchant site.                                                                                                                                                                |

> | `create-paymentorder-capture`     | `DEPRECATED DO NOT USE!` The second part of a two-phase transaction where the authorized amount is sent from the payer to the payee. It is possible to do a part-capture on a subset of the authorized amount. Several captures on the same payment are possible, up to the total authorization amount. |

Reviewer: ```suggestion
| `create-paymentorder-capture`     | The second part of a two-phase transaction where the authorized amount is sent from the payer to the payee. It is possible to do a part-capture on a subset of the authorized amount. Several captures on the same payment are possible, up to the total authorization amount. |
```

> | `create-paymentorder-cancel`      | `DEPRECATED DO NOT USE!` Used to cancel authorized and not yet captured transactions. If a cancellation is performed after doing a part-capture, it will only affect the not yet captured authorization amount.                                                                                         |

Reviewer: ```suggestion
| `create-paymentorder-cancel`      | Used to cancel authorized and not yet captured transactions. If a cancellation is performed after doing a part-capture, it will only affect the not yet captured authorization amount.                                                                                         |
```

> | `create-paymentorder-reversal`    | `DEPRECATED DO NOT USE!` Used to reverse a payment. It is only possible to reverse a payment that has been captured and not yet reversed.                                                                                                                                                               |

Reviewer: ```suggestion
| `create-paymentorder-reversal`    | Used to reverse a payment. It is only possible to reverse a payment that has been captured and not yet reversed.                                                                                                                                                               |
```

| `create-paymentorder-reversal`    | Used to reverse a payment. It is only possible to reverse a payment that has been captured and not yet reversed.                                                                                                                                                               |
| `paid-paymentorder`               | Returns the information about a paymentorder that has the status `paid`.                                                                                                                                                                                                       |
| `failed-paymentorder`             | Returns the information about a paymentorder that has the status `failed`.                                                                                                                                                                                                     |

> | `create-capture`                  | `DEPRECATED DO NOT USE!`                                                                                                                                                                                                                                                       |
> | `create-cancel`                   | `DEPRECATED DO NOT USE!`                                                                                                                                                                                                                                                       |
> | `create-cancel`                   | `DEPRECATED DO NOT USE!`                                                                                                                                                                                                                                                       |

Reviewer: ```suggestion
| `create-capture`                  | **Deprecated operation. Do not use!**                                                                                                                                                                                                                                                     |
| `create-cancel`                   | **Deprecated operation. Do not use!**                                                                                                                                                                                                                                                     |
| `create-cancel`                   | **Deprecated operation. Do not use!**                                                                                                                                                                                                                                                     |
```
